### PR TITLE
Dependencies management with Gradle buildSrcVersions

### PR DESCRIPTION
**Pull request by jmfayard**
## Description
Hello, I refactored the Gradle build to use my plugin buildSrcVersions to manage dependencies.

The files `{Libs,Versions}.kt` have been generated with the command 
  `$ ./gradlew buildSrcVersions`

See https://github.com/jmfayard/buildSrcVersions

See this article  https://dev.to/jmfayard/better-dependency-management-in-android-studio-3-5-with-gradle-buildsrcversions-34e9

## Next Steps

Currently the versions in the dependencies are exactly the same as on master. 

If you wish to update Kotlin, edit `Versions.kt`

If you want to update Gradle, run `$ ./gradlew wrapper`

### CODE REVIEW

**Comment by maestromac** [comment]
Thank you for this PR @jmfayard . We're definitely love any plugin that reduce headaches 😄 

I'm hitting this error
```
Task 'buildSrcVersions' not found in root project 'DEV-Android'.
```
I checked your installation doc, is this caused by not adding `buildSrcVersions` to plugin block?

**Comment by jmfayard** [comment]
@maestromac Correct! I don't know why this part didn't get commited. I don't have access to my development computer but hopefully I got the syntax right.

File: build.gradle.kts [hunk 1]
```diff
@@ -14,6 +14,10 @@ buildscript {
         // in the individual module build.gradle files
     }
 }
+plugins {
+  id("de.fayard.buildSrcVersions") version "0.4.2"
```
Comment (friederbluemle) [inline build.gradle.kts]: Please also indent using 4 spaces, like the rest of this and other Gradle files.

**Comment by jmfayard** [comment]
@friederbluemle done and I git-rebased to keep the history clean

**Pull request merged**